feature-group-definitions/font-size-adjust.dist.yml

```diff
@@ -0,0 +1,16 @@
+# Generated from: font-size-adjust.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: font-size-adjust
+description: The `font-size-adjust` CSS property preserves apparent text size, regardless of the font used, by scaling fonts to the same size with respect to a specific metric, such as x-height. This can help make fallback fonts look the same size.
+spec: https://drafts.csswg.org/css-fonts-5/#font-size-adjust-prop
+caniuse: font-size-adjust
+status:
+  baseline: false
+  support:
+    firefox: "3"
+    firefox_android: "4"
+    safari: "16.4"
+    safari_ios: "16.4"
+compat_features:
+  - css.properties.font-size-adjust
```

Comment:
The Firefox notes on this are _weird_:

> Before Firefox 40, <code>font-size-adjust: 0</code> was incorrectly interpreted as <code>font-size-adjust: none</code> (<a href="https://bugzil.la/1144885">bug 1144885</a>).

Fixed in version 40, but support for `none` came later in Firefox 72? This doesn't make sense. This makes me wonder whether `none` really was a later addition (and I ought to come up with a test case), or if I'm misunderstanding the meaning of the note and linked bug.

Reply:
Firefox 72 is because of https://github.com/mdn/browser-compat-data/issues/22931. We should treat all CSS property values in BCD as suspect until that is resolved.

Reply:
I'm fixing that in https://github.com/mdn/browser-compat-data/pull/22936.

Reply:
Oh yikes about the ranged values. Reviewed and merged https://github.com/mdn/browser-compat-data/pull/22936.

Reply:
That hasn't been released yet. I'll add a TODO to unblock this.